src/AWSMetadata.jl

```diff
@@ -20,6 +20,13 @@ using Retry
 
 github_headers = ["User-Agent" => "https://github.com/JuliaCloud/AWSCore.jl/blob/master/src/AWSMetadata.jl"]
```

Comment:
I doubt this matters but it should probably be changed:
```suggestion
github_headers = ["User-Agent" => "https://github.com/JuliaCloud/AWSAPI.jl/blob/master/src/AWSMetadata.jl"]
```